commands.go

```diff
@@ -170,3 +170,21 @@ func Sequentially(cmds ...Cmd) Cmd {
 		return nil
 	}
 }
+
+// setWindowTitleMsg is an internal message used to set the window title.
+type setWindowTitleMsg string
+
+// SetWindowTitle produces a command that sets the terminal title.
+// The message returned contains the title that was set.
```

Comment:
Actually, I'd drop this part of the comment because it doesn't pertain to the end user.